gpu_support/cuda_utils/install_cuda.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+install_cuda_version=$1
+cuda_install_dir=$2
```

Comment:
General CUDA installation is done via #212 now so I don't think you need this argument. This script is only about installing the CUDA package under `host_injections`...but changing the name of the script to reflect that is probably a good idea.